tests/python/pants_test/backend/python/tasks/BUILD

```diff
@@ -23,57 +23,276 @@ python_library(
 )
 
 python_tests(
-  sources=globs('test_*.py', exclude=([globs('*_integration.py')])),
-  dependencies=[
+  name = 'build_local_python_distributions',
+  sources = ['test_build_local_python_distributions.py'],
+  dependencies = [
+    '3rdparty/python:pex',
     '3rdparty/python/twitter/commons:twitter.common.collections',
-    '3rdparty/python/twitter/commons:twitter.common.dirutil',
-    '3rdparty/python:coverage',
+    'src/python/pants/backend/python:python_requirement',
+    'src/python/pants/backend/python/targets',
+    'tests/python/pants_test/backend/python/tasks/util',
+  ],
+)
+
+python_tests(
+  name = 'build_local_python_distributions_integration',
+  sources = ['test_build_local_python_distributions_integration.py'],
+  dependencies = [
+    'src/python/pants/util:collections',
+    'src/python/pants/util:contextutil',
+    'src/python/pants/util:process_handler',
+    'tests/python/pants_test:int-test',
+    'tests/python/pants_test/testutils:py2_compat',
+  ],
+  tags = {'integration'},
+  timeout=450
```

Comment:
This is horrific. It's true the LHS is even worse, which is ~incomprehensible. I look forward to the day where we can kill all timeout arguments in our BUILD files.

Reply:
Agreed :/ Part of the motivation for this PR is to call attention to which tests in particular take a long time.